src/checkout-buttons/strategies/amazon-maxo/amazon-maxo-button-strategy.spec.ts

```diff
@@ -156,8 +186,6 @@ describe('AmazonMaxoButtonStrategy', () => {
                     expect(button.firstChild).toBe(null);
                 }
             }
-
-            container = document.createElement('div');
             document.body.appendChild(container);
```

Comment:
You still keep this line, actually this is a behavior after the test complete the expect, so this line also needs to be either `beforeEach` or `afterEach`

Reply:
Yeah forgot to change it!, fixed ♻️